src/crypto/algorithms/megolm.js

```diff
@@ -657,7 +689,7 @@ MegolmDecryption.prototype._retryDecryption = function(senderKey, sessionId) {
 
     for (let i = 0; i < pending.length; i++) {
         try {
-            this.decryptEvent(pending[i]);
+            this.decryptEvent(pending[i], false);
```

Comment:
Wrong method call. Should be `this._decryptEvent`. Also, it would be good to explain why you're passing `false` here, presumably because we're retrying once we've got the keys, so getting the keys again isn't going to make a difference.